committeeoversightapp/models.py

```diff
@@ -194,7 +195,17 @@ def ratings_by_congress_desc(self):
 
     @property
     def ratings_by_congress_asc(self):
-        return self.committeerating_set.all().order_by('congress__id')
+        committeeratings = self.committeerating_set.all().order_by('congress__id')
+
+        count = 1
+        for rating in committeeratings:
+            if rating.congress.has_footnote:
```

Comment:
Clever. This feels a bit unconventional, but I'm having trouble thinking of an alternative.

Reply:
Yep I also spent a while trying to think of something that's more built into Django, but this is where I landed. If you think of another way later I'd be curious to hear it! I think this is ok for one instance on the site, but if we had this same problem for other types of data it would be worth abstracting somehow.